resources/Microsoft.Windows.Setting.Time/Microsoft.Windows.Setting.Time.psm1

```diff
@@ -39,57 +40,107 @@ function Get-ValidTimeZone {
 
 .DESCRIPTION
-    This `Time` DSC Resource allows you to manage the time zone, automatic time zone update, and system tray date/time visibility settings on a Windows machine.
+    This `TimeZone` DSC Resource allows you to manage the time zone, automatic time zone update, and system tray date/time visibility settings on a Windows machine.
 
-.PARAMETER TimeZone
-    The time zone to set on the machine. The value should be a valid time zone ID from the list of time zones (Get-TimeZone -ListAvailable).Id. The default value is the current time zone.
+.PARAMETER Id
+    The Id to set on the machine. The value should be a valid time zone ID from the list of time zones (Get-TimeZone -ListAvailable).Id. The default value is the current time zone.
 
 .PARAMETER SetTimeZoneAutomatically
-    Whether to set the time zone automatically. The value should be a boolean. You can find the setting in `Settings -> Time & Language -> Date & Time -> Set time automatically.
+    Whether to set the time zone automatically. The value should be a boolean. You can find the setting in `Settings -> Time & Language -> Date & Time -> Set time zone automatically.
+
+.PARAMETER SetTimeAutomatically
+    Whether to set the time automatically. The value should be a boolean. You can find the setting in `Settings -> Time & Language -> Date & Time -> Set time automatically.
 
 .PARAMETER AdjustForDaylightSaving
     Whether to adjust for daylight saving time. The value should be a boolean. You can find the setting in `Settings -> Time & Language -> Date & Time -> Adjust for daylight saving time automatically.
 
 .EXAMPLE
-    PS C:\> Invoke-DscResource -Name Time -Method Set -Property @{ TimeZone = "Pacific Standard Time"}
+    PS C:\> Invoke-DscResource -Name TimeZone -ModuleName Microsoft.Windows.Setting.Time -Method Set -Property @{ TimeZone = "Pacific Standard Time"}
 
     This example sets the time zone to Pacific Standard Time.
 
 .EXAMPLE
-    PS C:\> Invoke-DscResource -Name Time -Method Get -Property {}
+    PS C:\> Invoke-DscResource -Name TimeZone -ModuleName Microsoft.Windows.Setting.Time -Method Get -Property {}
 
     This example gets the current time settings on the machine.
 #>
 [DscResource()]
 class TimeZone {
     [DscProperty(Key)]
-    [string] $TimeZone
+    [string] $Id
 
     [DscProperty()]
     [nullable[bool]] $SetTimeZoneAutomatically
 
+    [DscProperty()]
+    [nullable[bool]] $SetTimeAutomatically
+
     [DscProperty()]
     [nullable[bool]] $AdjustForDaylightSaving
 
-    static hidden [string] $SetTimeZoneAutomaticallyProperty = 'Type'
+    static hidden [string] $SetTimeZoneAutomaticallyProperty = 'Start'
+    static hidden [string] $SetTimeAutomaticallyProperty = 'Type'
     static hidden [string] $AdjustForDaylightSavingProperty = 'DynamicDaylightTimeDisabled'
+    static hidden [string] $NtpEnabled = 'NTP'
+    static hidden [string] $NtpDisabled = 'NoSync'
+    static hidden [bool] $SupportsDaylightSavingProperty = $false
```

Comment:
This shouldn't be static as this value changes depending on the TimeZone instance. It would be better to have this as a nonconfigurable parameter that we determine in the Get() method. Here's an [example](https://github.com/microsoft/winget-dsc/blob/583d2c7671481e5d52e776e9f5af040e97fe51a0/resources/Microsoft.Windows.Developer/Microsoft.Windows.Developer.psm1#L59-L60)